src/vstest.console/Processors/EnableCodeCoverageArgumentProcessor.cs

```diff
@@ -105,6 +105,79 @@ internal class EnableCodeCoverageArgumentExecutor : IArgumentExecutor
 
         private static string codeCoverageCollectorSettingsTemplate =
```

Comment:
>codeCoverageCollectorSettingsTemplate [](start = 30, length = 37)

please have this content as an embedded resource instead of using Environment.NewLines all over the place